swagger.json

```diff
@@ -1519,6 +1573,52 @@
                 }
             }
         },
+        "/fleet/hos/audit_logs": {
+            "get": {
+                "tags": [
+                    "Fleet"
+                ],
+                "summary": "/fleet/hos/audit_logs",
+                "description": "Get the list of HOS (hours of service) audit logs for the specified time range.",
+                "operationId": "get_hos_audit_logs",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    { 
+                        "$ref": "#/parameters/groupIdParam" 
+                    },
+                    {
+                        "name": "start_ms",
+                        "description": "Beginning of the time range, specified in milliseconds Unix epoch time.",
+                        "required": true,
+                        "in": "query",
+                        "type": "integer"
+                    },
+                    {
+                        "name": "end_ms",
+                        "description": "End of the time range, specified in milliseconds Unix epoch time.",
```

Comment:
This is probably a widespread issue across all of our endpoints, but we should specify whether `start_ms` and `end_ms` are inclusive or exclusive.